from unittest import TestCase


class TestErrorHandler(TestCase):
    pass
